SCYLLA-VERSION-GEN

```diff
@@ -60,7 +60,7 @@ fi
 
 # Default scylla product/version tags
 PRODUCT=scylla
-VERSION=5.1.dev
+VERSION=5.1.0-dev
```

Comment:
A dash separates the version from the release. We should use ~ here so it's part of the version.

Reply:
@avikivity the problem is that `~` is not compatible with semantic version

```
[yaronkaikov@london]~/git/scylla-pkg (run-cloud-image-build-from-packer-dir)$ python
Python 3.10.5 (main, Jun  9 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import semver
>>> semver.parse('5.1.0~dev')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/semver.py", line 159, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/semver.py", line 190, in parse
    return VersionInfo.parse(version).to_dict()
  File "/usr/lib/python3.10/site-packages/semver.py", line 726, in parse
    raise ValueError("%s is not valid SemVer string" % version)
ValueError: 5.1.0~dev is not valid SemVer string
>>> semver.parse('5.1.0-dev')
OrderedDict([('major', 5), ('minor', 1), ('patch', 0), ('prerelease', 'dev'), ('build', None)])
>>> semver.parse('5.1.0')
OrderedDict([('major', 5), ('minor', 1), ('patch', 0), ('prerelease', None), ('build', None)])

```

Reply:
I see. What does scylla-operator use for the semver name? The rpm/deb package name? The output of scylla --version?